corehq/apps/app_manager/tests/test_apps.py

```diff
@@ -127,7 +127,7 @@ def test_undo_delete_app_removes_deleted_couch_doc_record(self):
         assert obj.doc_type.endswith("-Deleted"), obj.doc_type
         rec.undo()
         with self.assertRaises(DeletedCouchDoc.DoesNotExist):
-            DeletedCouchDoc.objects.get(doc_id=self.app._id, doc_type=self.app.doc_type)
+            DeletedCouchDoc.objects.get(doc_id=rec._id, doc_type=rec.doc_type)
```

Comment:
I think `rec.doc_type` is incorrect here since it will have switched back to the non-deleted version after `rec.undo()`, however the test will still pass because a `DeletedCouchDoc` record will not exist for `doc_type="Application"`. I think this should use a literal like `"Application-Deleted"` to avoid accidentally using the wrong query parameter. Alternately, use the slightly safer pattern in the next suggestion.

Reply:
`rec.doc_type` here refers to `DeleteApplicationRecord` not the actual application that's been deleted. Definitely confused me too since both Applications and Groups append '-Deleted' to the deleted objects' doc_type and feels like that should be used but like with all the other models that inherit `DeleteRecord` its the Delete(Model)Record's _id and doc_type that's used to create the DeleteCouchDoc